bazelbuild/bazel
# Windows, test wrapper: generate test XML output

In this commit:

- Add code to measure the duration the test ran
  for.

- Add code to write an XML file and to convert the
  test outerr log to CDATA-safe data.

Manual testing shows that the XML file is
identical to that produced by test-setup.sh

See https://github.com/bazelbuild/bazel/issues/5508
